exercises/pythagorean-triplet/description.md

```diff
@@ -17,6 +17,6 @@ For example,
 3**2 + 4**2 = 9 + 16 = 25 = 5**2.
 ```
 
-There exists exactly one Pythagorean triplet for which a + b + c = 1000.
+Given an input integer N, find all Pythagorean triplets for which `a + b + c = N`.
```

Comment:
Debatable: whether to explicitly note here that "none" is a possible answer. For example:

> all Pythagorean triplets for which `a + b + c = N` (possibly zero triplets)

I chose not to since…

* The additional clarification is not necessary for this to be a true statement. Zero triplets is a valid result for the word "all".
* Even if a student were to miss the possibility of zero triplets, it is highly unlikely to be a fatal error requiring a rewrite of the entire implementation upon discovery. The student has already been dealing with a variable number of triplets because the test case for zero triplets comes after the test cases for multiple triplets. Even in a language whose type system prevents the return type from having zero elements, in all such languages I know, making that change is a mere one-character change (change a + to a * or something similarly easy)

Reply:
EBWODP:

> in all such languages I know

I'll leave it less ambiguous as to how many languages I know and instead say "in the [only such language](https://ceylon-lang.org/documentation/1.3/tour/sequences/) I know"